tests/test_cli.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: MIT
+
+import sys
+import unittest
+from argparse import Namespace
+
+import odxtools.cli.find as find
+import odxtools.cli.list as list_tool
+
+import_failed = False
+
+try:
+    import odxtools.cli.browse as browse
+except:
+    import_failed = True
```

Comment:
Why are you doing a try/except ?

Reply:
that's my fault: on Linux systems the `PyInquirer` module often does not work due to import errors. If this happens, we do not want the unit tests to fail...

Reply:
@andlaus consider switching to https://rich.readthedocs.io/en/stable/prompt.html